ios/FluentUI/Button/FluentButtonToggleStyle.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+
+/// `ToggleStyle` which configures the `Toggle` according to its state and design tokens.
+public struct FluentButtonToggleStyle: ToggleStyle {
+    public init() {}
+
+    @Environment(\.fluentTheme) private var fluentTheme: FluentTheme
+
+    public func makeBody(configuration: Configuration) -> some View {
+        SwiftUI.Button(action: {
+            configuration.isOn.toggle()
+        }, label: {
+            configuration.label
+        })
+        .buttonStyle(configuration.isOn ? buttonStylePressed : buttonStyle)
```

Comment:
In this case does `pressed` refer to the button being toggled `ON` or that the button is being touched/pressed by the user? If it's the former, my concern is that the use of `pressed` might get confused with the use of `pressed` in the base `FluentButtonStyle` where it means the button is being touched/pressed by the user. Using a different adjective might like `selected` or `toggled` might help remove this possible confusion.

Reply:
Good catch, I'll change the name to match the `configuration` property `isOn`.